remoteappmanager/db/orm.py

```diff
@@ -0,0 +1,142 @@
+import contextlib
+
+from sqlalchemy import Column, Integer, Boolean, Unicode, ForeignKey
+from sqlalchemy.exc import OperationalError
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, relationship
+from sqlalchemy import create_engine
+
+from remoteappmanager.logging.logging_mixin import LoggingMixin
+
+Base = declarative_base()
+
+
+class UserTeam(Base):
+    """ The user (n <-> n) team association table """
+    __tablename__ = "user_team"
+    id = Column(Integer, primary_key=True)
+    user_id = Column(Integer, ForeignKey('user.id'))
+    team_id = Column(Integer, ForeignKey('team.id'))
```

Comment:
From the examples that I saw that implement many-to-many mappings assigned primary_key=True to both of these ids (`user_id` and `team_id` in this case).  There is no need for an additional `id` for the table either.  (e.g. https://github.com/jupyterhub/jupyterhub/blob/master/jupyterhub/orm.py#L262)